src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -414,6 +414,16 @@ cdef class Font:
         else:
             return None
 
+    def get_glyph_extents(self, gid: int):
+        cdef hb_glyph_extents_t extents
+        success = hb_font_get_glyph_extents(self._hb_font, gid, &extents)
+        if success:
+            return (extents.x_bearing, extents.y_bearing,
+                    extents.width, extents.height)
```

Comment:
It would have nicer if the function returned a namedtuple, having to guess which value is which Everytime reading code using this function is a bit unfortunate.

Reply:
You're right, will fix.